roles/uEsimulator/defaults/main.yml

```diff
@@ -0,0 +1,12 @@
+srsran:
+  docker:
+    container:
+      ue_image: srsran-ue:latest
```

Comment:
Should you update this? Such image does not exist, correct?

Reply:
done